src/schemaPostgres.ts

```diff
@@ -10,6 +10,9 @@ import {
 
 const pgp = PgPromise();
 
+/**
+ * Converts a postgres data type to a typescript
```

Comment:
I think this is already clear from the function name :)

Reply:
Agreed!